### DISCUSSION TITLE
When will #7541 be merged to `master` / released?

### DISCUSSION THREAD

**Discussion by HazyFish** [firebase/firebase-js-sdk#7688] [General]
We need to revoke token when the user deletes an account associated with Apple ID as part of Apple's requirement. This feature is already available on iOS SDK and we already have a merged PR for the JS SDK #7541. When will it be merged to `master` / released?
@NhienLam

**Comment by NhienLam** [comment]
Hi @HazyFish.
The changes are not ready to be merged and released yet. But we aim to release it soon. Thanks!
